package.json

```diff
@@ -81,4 +81,4 @@
       "path": "cz-conventional-changelog"
     }
   }
-}
+}
```

Comment:
I'm adding vsc workspace settings to automatically create new lines here, in the meantime, can you set it to add newlines?